src/ingest_validation_tools/submission.py

```diff
@@ -61,7 +63,7 @@ def get_errors(self):
         if reference_errors:
             errors['Reference Errors'] = reference_errors
 
-        if not tsv_errors and not reference_errors:
+        if self.plugin_dir_abs_path:
```

Comment:
```suggestion
        if self.plugin_dir_abs_path and not tsv_errors and not reference_errors:
```
I won't block on this, but I would prefer to keep the original checks, and I'd like to make #399 a priority, which I think would address your root problem. The problem I see is that there are a range of ways that submissions can be malformed, and these aren't "unexpected"... catching them is what this script does. The plugin checks just are more sensitive, and running them when there are more basic problems is going to cause problems.

Reply:
The issue is that I need to be able to *force* the plugin errors to be checked even in the presence of tsv and/or reference errors.  Would you rather have a different mechanism to force it from the command line?